less/build_vizor2teaser.less

```diff
@@ -0,0 +1,391 @@
+    padding-bottom: 100px;
+    background-color: @canvas-color-2;
+    @media (min-width: @large-viewport-width) {
+        padding-top: 160px;
+        padding-bottom: 160px;
+    }
+}
+.register__form {
+    margin-top: 35px;
+}
+.register__email {
+    width: 100%;
+}
+.register__email::placeholder {
+    color: inherit;
+}
+.register__hint {
+    margin-top: 14px;
+    max-width: 17em;
+    font-size: 10px;
+    color: @secondary-color;
+    @media (min-width: @medium-viewport-width) {
+        font-size: 12px;
+        max-width: none;
+    }
+}
+.register__submit {
+    margin-left: -4px; // Align optically
+}
+@media (max-width: (@large-viewport-width - 1px)) {
+    .register__submit {
+        margin-top: 27px;
+    }
+}
+@media (min-width: @large-viewport-width) {
+    .register__form {
+        display: flex;
+        justify-content: flex-start;
+        align-items: top;
+        margin-top: 100px;
+    }
+    .register__email-container {
+        flex: 0 0 50%;
+    }
+    .register__submit {
+        flex: 0 0 auto;
+        margin-left: 55px;
+    }
+}
+
+.button {
+    box-sizing: border-box;
+    width: 160px;
+    height: @input-height-small;
+    text-align: center;
+    font-size: inherit;
+    color: #0E6491;
+    background: transparent;
+    border: 2px solid @primary-color;
+    border-radius: 100px;
+    cursor: pointer;
+    outline: none;
+    -webkit-appearance: none;
+    @media (min-width: @medium-viewport-width) {
+        width: 184px;
+        height: @input-height-medium;
+    }
+}
+.button:hover {
+    color: @accent-color;
+    border-color: @accent-color;
+}
+.button:active {
+    background-color: @accent-color;
+    border-color: @accent-color;
+    color: #FFFFFF;
+}
+
+.text-input {
+    box-sizing: border-box;
+    height: @input-height-small;
+    padding: @input-padding-small 0;
+    font-size: 16px;
+    line-height: @input-height-small - @input-padding-small;
+    color: inherit;
+    background: transparent;
+    border: none;
+    border-bottom: 2px solid @primary-color;
+    outline: none;
+    -webkit-appearance: none;
+    @media (min-width: @medium-viewport-width) {
+        height: @input-height-medium;
+        padding-top: @input-padding-medium;
+        padding-bottom: @input-padding-medium;
+        line-height: @input-height-medium - @input-padding-medium;
+        font-size: 20px;
+    }
+}
+.text-input:hover {
+    border-bottom-color: @accent-color;
+}
+.text-input:focus {
+    border-bottom-color: @accent-color;
+}
+.text-input:focus::placeholder {
+    color: @secondary-color;
+}
+
+.footer {
+    background-color: @canvas-color-2;
+}
+.footer__wrapper {
+    padding-top: 50px;
+    padding-bottom: 50px;
+    border-top: 1px solid #C4C4C4;
+}
+.footer__link-list {
+    font-size: 20px;
+    line-height: 40px;
+}
+.footer__info {
+    margin-top: 60px;
+    strong {
+        font-weight: bold;
+    }
+    a {
+        text-decoration: underline;
+    }
+    a:hover {
```

Comment:
you can nest these as `&:hover { ... }`, `&:active { ... }` etc inside the `a { }`.